appshell/update_win.cpp

```diff
@@ -54,7 +54,7 @@ void UpdateHelper::RunAppUpdate() {
 		std::wstring commandInput = L"msiexec /i " + m_InstallerPath + L" /qr";
 		//AUTOUPDATE_PRERELEASE
 		if (!m_logFilePath.empty()) {
-			commandInput += L" /l*V " + m_logFilePath;
+			commandInput += L" /li " + m_logFilePath;
```

Comment:
@niteskum , please verify the success or failure log message in the logs, if it changes because of this change.
Also, roughly, what is the impact of this change on the performance of updation?

Reply:
There in no change in Log Error or Success Message.
Performance improvement in installation is of around 10 secs.
But Log Size has reduced to 15 KB from 50 MB.